ci/ci_steps.sh

```diff
@@ -37,8 +37,11 @@ else
     cp -f "$ENVOY_BUILD_DIR"/envoy/source/exe/envoy ./build_release
     # This script builds a lyft/envoy image and pushes that image on merge to master.
     ./ci/docker_push.sh
-    # This script runs on every PRs release run to test the docker examples.
-    ./ci/verify_examples.sh
+    if [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ "$TRAVIS_BRANCH" == "master" ]
```

Comment:
nit: I was mainly commenting that we have guarding in multiple places, which is confusing. Can we just guard in this file and not in the called scripts? Or just there?